beacon-chain/core/peerdas/info.go

```diff
@@ -107,3 +107,106 @@ func computeInfoCacheKey(nodeID enode.ID, custodyGroupCount uint64) [nodeInfoCac
 
 	return key
 }
+
+// ColumnIndices is a map of column indices where the key is the column index and the value is a boolean.
+// The boolean could indicate different things, eg whether the column is needed (in the context of satisfying custody requirements)
+// or present (in the context of a custody check on disk or in cache).
+type ColumnIndices map[uint64]struct{}
```

Comment:
Is it thread safe? (Seems not).
Could it be either thread safe, or in the contrary could a warning in the godoc indicate it is not?
Eg in the [stdlib](https://cs.opensource.google/go/go/+/refs/tags/go1.25.3:src/math/rand/rand.go;l=31).

Because `ColumnIndices` is publicly exposed, the user may think it is thread safe by default.